tests/app/test_utils.py

```diff
@@ -0,0 +1,55 @@
+from app.utils import (display_list,
+                       strip_whitespace_from_data,
+                       json_has_matching_id,
+                       json_has_required_keys,
+                       link)
+
+from nose.tools import assert_equal, raises
```

Comment:
API was switched to pytest, so we should probably start using pytest tools for new tests